spec/tools/database_spec.rb

```diff
@@ -1,3 +1,4 @@
+require_relative '../spec_helper'
```

Comment:
I believe this line is unnecessary. The tests run well without it. Can you double check?

Reply:
This can really be removed with no side effects.